test/pytest/test_torch_export.py

```diff
@@ -123,6 +123,6 @@ def test_torch_export_aot_compile_dynamic_batching(custom_working_directory):
         data["body"] = byte_array_type
 
     # Send a batch of 16 elements
-    result = handler.handle([data for i in range(15)], ctx)
+    result = handler.handle([data for i in range(32)], ctx)
```

Comment:
how about parameterizing the test and test both?

Reply:
Thanks for catching this. I changed it